shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/main/java/org/apache/shardingsphere/encrypt/rewrite/token/generator/impl/EncryptProjectionTokenGenerator.java

```diff
@@ -88,6 +111,90 @@ protected boolean isGenerateSQLTokenForEncrypt(final SQLStatementContext sqlStat
         return result;
     }
     
+    private Map<String, SimpleTableSegment> distinct(final Collection<SimpleTableSegment> simpleTableSegments) {
```

Comment:
@cheese8 Does this method just for distinct? If so, you can use `TablesContext.uniqueTables`.

Reply:
yes, thanks for hints